doc/getting_started/setup_fpga.md

```diff
@@ -357,7 +357,7 @@ Then a connection between OpenOCD and GDB may be established with:
 ```console
 cd $REPO_TOP
 riscv32-unknown-elf-gdb -ex "target extended-remote :3333" -ex "info reg" \
-  $(find -L bazel-out/ -type f -name "uart_smoketest_sim_verilator | head -n 1")
+  $(find -L bazel-out/ -type f -name "uart_smoketest_prog_fpga_cw310.elf" | head -n 1)
```

Comment:
This `find` command substitution is a little funky. WDYT about using `bazelisk.sh`'s `outquery`?

```
$(./bazelisk.sh outquery --config=riscv32 //sw/device/tests:uart_smoketest_prog_fpga_cw310.elf)
```

Reply:
I think you will actually need
```
$(./bazelisk.sh outquery --config=riscv32 --ui_event_filters=-info --noshow_progress //sw/device/tests:uart_smoketest_prog_fpga_cw310.elf)
```
to achieve the same effect as the find command. 

For consistency, perhaps it might be better to keep the corrected `find` command in this PR, and then in a separate PR update the `./bazelisk.sh outquery ...` command to always hide the progress messages (by adding the `--ui_event_filters=-info --noshow_progress` flags) and then update the
- build_sw.md
- setup_fpga.md
- setup_verilator.md

docs to all use the `./bazelisk.sh outquery` command, since they all use the `find` command right now.

Thoughts @dmcardle ?

Reply:
Good idea(s)! I was going to suggest tacking on `2>/dev/null` inside the command substitution to keep it short, but I think that your suggestion is better.

I won't block this PR on the `outquery` suggestion.

Filed #14806 to follow up on this.

Reply:
Thanks @dmcardle and @timothytrippel, I'll go ahead and merge this as is in that case!